src/legacy/core_plugins/kbn_doc_views/public/views/table.html

```diff
@@ -48,35 +51,41 @@
         </span>
         <span ng-if="!indexPattern.metaFields.includes(field) && !mapping[field].scripted">
           <button
-            aria-label="Filter for field present"
+            aria-label="{{ ::'kbnDocViews.table.filterForFieldPresentButtonAriaLabel' | i18n: { defaultMessage: 'Filter for field present' } }}"
             ng-click="filter('_exists_', field, '+')"
-            tooltip="Filter for field present"
+            tooltip="{{ ::'kbnDocViews.table.filterForFieldPresentButtonTooltip' | i18n: { defaultMessage: 'Filter for field present' } }}"
             tooltip-append-to-body="1"
           >
             <i
               class="fa fa-asterisk"></i>
           </button>
         </span>
-        <span ng-if="indexPattern.metaFields.includes(field)" tooltip="Unable to filter for presence of meta fields">
+        <span
+          ng-if="indexPattern.metaFields.includes(field)"
+          tooltip="{{ ::'kbnDocViews.table.unableToFilterForPresenceOfMetaFieldsTooltip' | i18n: { defaultMessage: 'Unable to filter for presence of meta fields' } }}"
+        >
           <i class="fa fa-asterisk text-muted"></i>
         </span>
-        <span ng-if="mapping[field].scripted" tooltip="Unable to filter for presence of scripted fields">
+        <span
+          ng-if="mapping[field].scripted"
+          tooltip="{{ ::'kbnDocViews.table.unableToFilterForPresenceOfScriptedFieldsTooltip' | i18n: { defaultMessage: 'Unable to filter for presence of scripted fields' } }}"
+        >
           <i class="fa fa-asterisk text-muted"></i>
         </span>
       </td>
 
       <td>
         <i ng-if="!mapping[field] && field[0] === '_'"
           tooltip-placement="top"
-          tooltip="Field names beginning with _ are not supported"
+          tooltip="{{ ::'kbnDocViews.table.fieldNamesBeginningWithUnderscoreUnsupportedTooltip' | i18n: { defaultMessage: 'Field names beginning with _ are not supported' } }}"
```

Comment:
**suggestion:** I'd move `_` into `values` since this symbol should not be changed after translation.